tests/evaluate/test_default_evaluator.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Can we add a test case for classification with text labels and make sure it works?

Reply:
Added!